src/ComponentStatus.mdx

```diff
@@ -0,0 +1,124 @@
+---
+name: Component Status
+---
+
+import { Table } from "react-bootstrap";
+
+# Status
+
+The Blue Design System team is working diligently to build and perfect components to suit your design needs. Check this page to see the status of the most sought after components. If you don’t find what you’re looking for, you can contact us at [bluedesignsystem@endurance.com](mailto:bluedesignsystem@endurance.com).
+
+<Table>
```

Comment:
We should avoid using our actual components for our documentation and instead rely on the markdown components from Docz so we can theme our tables separately from the `uaf-design-system` tables. https://www.markdownguide.org/extended-syntax/#tables

- [ ] convert to md table